src/containers/Transactions/DetailTab/Description.tsx

```diff
@@ -29,6 +30,13 @@ export const TransactionDescription: FC<{ data: any }> = ({ data }) => {
           />
         </b>
       </div>
+      {data.tx.Delegate && (
+        <div data-testid="delegate">
+          {t('tx_delegated_to')}
+          {` `}
+          <Account account={data.tx.Delegate} />
```

Comment:
This could be simplified using the react-i18n feature that allows you to insert in a React component. I don't remember how to do it off the top of my head but I think it's in parts of #973. Easier to handle syntax from other languages that aren't SVO as well that way.

Ditto for all the other translation strings

Reply:
Great call! I will make the changes